website/content/api-docs/secret/databases/index.mdx

```diff
@@ -92,13 +96,14 @@ are supported and any additional details about them.
   "allowed_roles": "readonly",
   "connection_url": "{{username}}:{{password}}@tcp(127.0.0.1:3306)/",
   "username": "vaultuser",
-  "password": "secretpassword"
+  "password": "secretpassword",
+  "disable_escaping": "false"
```

Comment:
The default is false so this is redundant. Also this seems like a more "advanced" feature to me so it might not be something we want to include on this page. I view these examples as sort of the "bare minimum" a user needs to get rolling. I think adding this _might_ be a bit of a distraction?

Reply:
Right, I agree with this especially since I added an example further down in the file.